state.go

```diff
@@ -20,11 +20,23 @@ type namespace struct {
 	ClientKey            string `yaml:"clientKey"`
 }
 
+// config type represents the settings fields
+type config struct {
```

Comment:
Why the struct name is `config` and not `settings`?

Reply:
It was to make sure it wasn't going to conflict with any other existing variable. I can change it but, it is that important?

Reply:
Not at all, that fine as is.